doc/source/conf.py

```diff
@@ -257,5 +256,5 @@ def linkcode_resolve(domain, info):
 
     fn = relpath(fn, start=dirname(nmrglue.__file__))
 
-    return "http://github.com/jjhelmus/nmrglue/blob/master/nmrglue/%s%s" % (
+    return "http://github.com/jjhelmus/nmrglue/blob/master/nmrglue/{}{}".format(
```

Comment:
Could this and similar changed be moved to f-string based interpolation?

Such a change would require Python 3.6+ which at this point should be fine.

Reply:
I'm also not sure on how pyupdate chooses the format but the remaining `.format(...)` patterns all include involved parameters expressions.  Keeping those as-is seems the best approach.

Reply:
Fixed except:
* `doc/source/tutorial.rst`: this is a case where the documentation might be less readable
* `nmrglue/fileio/fileiobase.py`: should this vendored file be fixed upstream?

Reply:
I'll open a new PR for the cases where removing .format() seems feasible.